src/System.Net.HttpListener/tests/HttpListenerTimeoutManagerTests.cs

```diff
@@ -0,0 +1,359 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace System.Net.Tests
+{
+    [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
Timeouts throw PNSE on Unix currently. There's a test class below for Unix on this same file.